invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/locations.html

```diff
@@ -0,0 +1,62 @@
+{#
+Copyright (C) 2023 CERN.
+
+Invenio-app-rdm is free software; you can redistribute it and/or modify
+it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% macro render_point(coordinates) %}
+  {% if coordinates | length == 2 %}
+    (<a href="https://google.com/maps/place/{{coordinates[1]}},{{coordinates[0]}}">{{ coordinates[1] }}, {{
```

Comment:
This was taken from legacy. 

However, it can also be serialized in the backend and we could even configure other map providers.